>Integration tests (asan) [1/6] — fail: 2, passed: 393

- `test_zookeeper_config/test.py::test_chroot_with_same_root`

```
E           subprocess.TimeoutExpired: Command '['docker-compose', '--env-file', '/ClickHouse/tests/integration/test_zookeeper_config/_instances_0/.env', '--project-name', 'roottestzookeeperconfig', '--file', '/compose/docker_compose_keeper.yml', '--verbose', 'up', '-d']' timed out after 300 seconds
```

--- a/docker/test/integration/runner/compose/docker_compose_minio.yml
+++ b/docker/test/integration/runner/compose/docker_compose_minio.yml
@@ -2,9 +2,7 @@ version: '2.3'
 
 services:
   minio1:
-    # Newer version of minio results in such errors:
-    # "AWSErrorMarshaller: Encountered AWSError 'InternalError': We encountered an internal error, please try again"
-    image: minio/minio:RELEASE.2021-09-23T04-46-24Z
+    image: minio/minio:RELEASE.2023-09-30T07-02-29Z
     volumes:
       - data1-1:/data1
       - ${MINIO_CERTS_DIR:-}:/certs

--- a/tests/integration/test_storage_delta/test.py
+++ b/tests/integration/test_storage_delta/test.py
@@ -274,7 +274,7 @@ def test_multiple_log_files(started_cluster):
     assert len(files) == 2  # 1 metadata files + 1 data file
 
     s3_objects = list(
-        minio_client.list_objects(bucket, f"/{TABLE_NAME}/_delta_log/", recursive=True)
+        minio_client.list_objects(bucket, f"{TABLE_NAME}/_delta_log/", recursive=True)
     )
     assert len(s3_objects) == 1
 
@@ -288,7 +288,7 @@ def test_multiple_log_files(started_cluster):
     assert len(files) == 4  # 2 metadata files + 2 data files
 
     s3_objects = list(
-        minio_client.list_objects(bucket, f"/{TABLE_NAME}/_delta_log/", recursive=True)
+        minio_client.list_objects(bucket, f"{TABLE_NAME}/_delta_log/", recursive=True)
     )
     assert len(s3_objects) == 2
 

